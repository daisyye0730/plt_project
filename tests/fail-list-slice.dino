int main() {
  List(int, 4) li;
  List(int, 2) subli;

  li = [1, 2, 3, 4];
  subli = li[1:5];
  
}