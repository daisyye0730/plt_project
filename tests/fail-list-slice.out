Fatal error: exception Failure("index out of range for slicing in li")
